src/configSchema.ts

```diff
@@ -126,6 +127,7 @@ export const VALIDATION_SCHEMA = z
 		dataDirs: z.array(z.string()).nullish(),
 		matchMode: z.nativeEnum(MatchMode),
 		linkCategory: z.string().nullish(),
+		crossSeedTag: z.string().nullish(),
```

Comment:
As I said, I am not overly excited about this option name given the only client actually have a "Tags" feature is qbit, and the rest are called labels.